packages/client/src/helpers.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/no-named-as-default
+import Long from 'long';
```

Comment:
Please put the eslint directive comment on the same line as the `import`, like it's done in schedule-helpers.ts. Otherwise, these comments sometime get broken by tools that sort imports.

Reply:
I don't need it anymore...